modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,79 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
```

Comment:
The current and target releases have been mirrored to the internal registry is also a prerequisite

Reply:
done